protos/sources/data-node/api/v2/trading_data.proto

```diff
@@ -1766,29 +1767,29 @@ message ListERC20MultiSigSignerAddedBundlesRequest {
 
 // Response from adding a signature bundle to the signer list of a multisig contract for a particular validator
 message ListERC20MultiSigSignerAddedBundlesResponse {
-    // List of bundles for that validator, it may have been added multiple times if removed in between.
+    // Page of bundles for that validator - it may have been added multiple times if removed in between - and corresponding page information.
     ERC20MultiSigSignerAddedConnection bundles = 1;
 }
 
-// Represents the signature data that is to be added with the corresponding cursor for it
+// Signature data that is to be added with the corresponding cursor for it
 message ERC20MultiSigSignerAddedEdge {
     // Signature data to be added.
     vega.events.v1.ERC20MultiSigSignerAdded node = 1;
     // Cursor that can be used to fetch further pages.
     string cursor = 2;
 }
 
-// Represents the signature bundle data that is to be added the corresponding cursor for it
+// Signature bundle data that is to be added the corresponding cursor for it
```

Comment:
```suggestion
// Signature bundle data that is to be added and the corresponding cursor.
```
Please check this suggestion, the original sentence doesn't read well but my suggestion may not be accurate?